docker/docker-compose.yml

```diff
@@ -135,8 +139,8 @@ services:
       - source: proxy.key
         target: /etc/nginx/certs/rengine_rsa.key
     volumes:
-      - ./config/nginx/rengine.conf:/etc/nginx/conf.d/rengine.conf:ro
-      - ./web/staticfiles:/home/rengine/app/staticfiles/
+      - ./proxy/config/rengine.conf:/etc/nginx/conf.d/rengine.conf:ro
+      - ../web/staticfiles:/home/rengine/rengine/staticfiles/
```

Comment:
```suggestion
      - ../web/static:/home/rengine/rengine/staticfiles/
```
Wrong path, which causes the following (no markup):

![image](https://github.com/user-attachments/assets/2ec5b853-5805-4f14-8283-f31b2ad7b88a)

---

Either rename [the folder](https://github.com/Security-Tools-Alliance/rengine-ng/tree/install-prebuilt-build/web/static) or apply my fix.